contrib_bots/lib/xkcd.py

```diff
@@ -0,0 +1,120 @@
+import requests
+import logging
+from random import randint
+
+class xkcdHandler(object):
```

Comment:
Conventionally class names should be in camel case:

https://www.python.org/dev/peps/pep-0008/#class-names